src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/OssArtifactVersionScore.java

```diff
@@ -0,0 +1,104 @@
+package com.sap.oss.phosphor.fosstars.model.score.oss;
+
+import com.sap.oss.phosphor.fosstars.model.Score;
+import com.sap.oss.phosphor.fosstars.model.Value;
+import com.sap.oss.phosphor.fosstars.model.score.WeightedCompositeScore;
+import com.sap.oss.phosphor.fosstars.model.value.ScoreValue;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.oss.phosphor.fosstars.model.weight.ImmutableWeight;
+import com.sap.oss.phosphor.fosstars.model.weight.ScoreWeights;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * <p>The artifact version score evaluates how the given version relates to
+ * released versions, the release history of artifacts and if there are
+ * known vulnerabilities for the given artifact version
+ * of an open-source project.
```

Comment:
The name of this scoring functions says "Security score ..." but the description and javadoc doesn't mention it. Please update the descriptions or the name. It would be good to to start with a short description like "This scoring functions assesses ...". The, you can put more details about how the score is calculated.

Reply:
Hard to find a good name. 
The focus on the score is security as it checks e.g. vulnerability and this with the highest impact.
But the other sub-scores are only indirect related to security.
Hence I would keep it with _Security score for an artifact version of an open-source project_.

And in the javadoc and description security is mentioned with the vulnerabilities, or is this not enough?

Reply:
Not really, I think the description needs to be re-written. It would be good to to start with a short description like "This scoring functions assesses ...". The, you can put more details about how the score is calculated.

Reply:
The class name should mention "security" as well.